src/psyclone/tests/domain/lfric/lfric_builtins_test.py

```diff
@@ -4691,3 +4693,19 @@ def mesh_code_present(field_str, code):
         "      max_halo_depth_mesh = mesh%get_halo_depth()\n"
         "      !\n")
     assert output_dm_1 in code
+
+
+def test_field_arrays_in_builtins():
```

Comment:
I've renamed it. The other question already gave me a headache, since I originally considered having this test in the `var_info_test.py` file. Reason I put this here is that this test triggers the new lines in `domain.lfric.lfric_builtins.py`, and I think we are trying to get all lines of a file covered by running the corresponding tests (though afaik that's not yet part of the official coding style).  Admittedly, `lfric_builtin.py` is not yet fully covered, but leaving this test in the `lfric_builtins_test.py` file certainly helps.

Having said that, I actually don't mind (though I did find it useful that running the corresponding test at least covers 100% of the lines)